src/main/scala/com/kinja/amqp/AmqpConsumer.scala

```diff
@@ -224,7 +224,7 @@ class Listener[A: Reads](
 			originalSender ! ack
 			context.become(idle)
 		case Delivery(_, envelope, _, _) =>
-			sender ! Reject(envelope.getDeliveryTag, requeue = true)
+			originalSender ! Reject(envelope.getDeliveryTag, requeue = true)
```

Comment:
I don't understand this. If a message comes in while we are in a sleeping stage — why don't we send a rejection notice to the one who sent it?